emcc.py

```diff
@@ -789,19 +808,12 @@ def detect_fixed_language_mode(args):
             assert key != 'WASM_BACKEND', 'do not set -s WASM_BACKEND, instead set EMCC_WASM_BACKEND=1 in the environment'
       newargs = [arg for arg in newargs if arg is not '']
 
-      # Handle aliases in settings flags. These are settings whose name
-      # has changed.
-      settings_aliases = {
-          'BINARYEN': 'WASM',
-          'BINARYEN_MEM_MAX': 'WASM_MEM_MAX',
-          # TODO: change most (all?) other BINARYEN* names to WASM*
-      }
       settings_key_changes = set()
 
       def setting_sub(s):
         key, value = s.split('=', 1)
         settings_key_changes.add(key)
-        return '='.join([settings_aliases.get(key, key), value])
+        return '='.join([key, value])
```

Comment:
This function just returns its first argument now.  Might be worth removing it and replacing the list comprehension below with a loop?